<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pilotWindow</class>
 <widget class="QMainWindow" name="pilotWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>927</width>
    <height>583</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="cursor">
   <cursorShape>ArrowCursor</cursorShape>
  </property>
  <property name="mouseTracking">
   <bool>true</bool>
  </property>
  <property name="focusPolicy">
   <enum>Qt::StrongFocus</enum>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::ActionsContextMenu</enum>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>pilotWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="zasoby.qrc">
    <normaloff>:/ikony/games_256.png</normaloff>:/ikony/games_256.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgba(107, 107, 107, 231);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="up_area">
      <property name="spacing">
       <number>1</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetFixedSize</enum>
      </property>
      <item>
       <widget class="QLCDNumber" name="gazLCD_y">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(0, 255, 0, 50);</string>
        </property>
        <property name="digitCount">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="gazLCD_x">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(0, 255, 0, 50);</string>
        </property>
        <property name="digitCount">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="responsTxt">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(0, 255, 0, 50);
border: true;
border-color: rgb(0, 255, 0,255);
color: rgb(0, 255, 0);
font-family:  &quot;Franklin Gothic Medium&quot;;
font-size:  40px;</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="skretLCD_x">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(0, 255, 0, 50);</string>
        </property>
        <property name="digitCount">
         <number>4</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLCDNumber" name="skretLCD_y">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Fixed" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(0, 255, 0, 50);</string>
        </property>
        <property name="digitCount">
         <number>4</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="GamePad">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>gamePad</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="1" column="0" colspan="2">
         <widget class="QGraphicsView" name="graphicsView_gaz">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustToContents</enum>
          </property>
         </widget>
        </item>
        <item row="1" column="2">
         <layout class="QVBoxLayout" name="verticalLayout">
          <property name="spacing">
           <number>0</number>
          </property>
          <item>
           <widget class="myButton" name="buttonAutomatGearbox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Automat</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="myButton" name="push_plusGear">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(0, 255, 0, 50);
 border: true;
	border-color: rgb(0, 255, 0);
	color: rgb(0, 255, 0);
    font-family:  &quot;Franklin Gothic Medium&quot;;
    font-size:  40px;</string>
            </property>
            <property name="text">
             <string>Up</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLCDNumber" name="gear">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>2</pointsize>
             </font>
            </property>
            <property name="digitCount">
             <number>1</number>
            </property>
            <property name="intValue" stdset="0">
             <number>1</number>
            </property>
           </widget>
          </item>
          <item>
           <widget class="myButton" name="push_minusGear">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">background-color: rgba(0, 255, 0, 50);
 border: true;
	border-color: rgb(0, 255, 0);
	color: rgb(0, 255, 0);
    font-family:  &quot;Franklin Gothic Medium&quot;;
    font-size:  40px;</string>
            </property>
            <property name="text">
             <string>Down</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="3" colspan="2">
         <widget class="QGraphicsView" name="graphicsView_skret">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Ignored" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="verticalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="horizontalScrollBarPolicy">
           <enum>Qt::ScrollBarAlwaysOff</enum>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustToContents</enum>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="camera">
       <attribute name="title">
        <string>camera</string>
       </attribute>
       <widget class="QPushButton" name="pushButton">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>20</y>
          <width>861</width>
          <height>311</height>
         </rect>
        </property>
        <property name="text">
         <string>PushButton</string>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QGridLayout" name="down_area">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinAndMaxSize</enum>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0" colspan="3">
       <layout class="QHBoxLayout" name="reset_Layout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QCheckBox" name="checkBoxPower">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>powrot gazu na neutralna pozycje</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="tristate">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="myButton" name="reset">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(0, 255, 0, 50);
 border: true;
	border-color: rgb(0, 255, 0);
	color: rgb(0, 255, 0);
    font-family:  &quot;Franklin Gothic Medium&quot;;
    font-size:  40px;</string>
          </property>
          <property name="text">
           <string>RESET</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBoxWheel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(255, 255, 255);</string>
          </property>
          <property name="text">
           <string>powrot kierownicy na neutralna pozycje</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <property name="tristate">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="myTouchSlider" name="horizontalSlider">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Ignored">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QSlider::groove:horizontal {
border: 6px solid #AAA;
height: 10px;
background: #9AD9EB;
margin: 2px 0;
}

QSlider::handle:horizontal {
 
	background-color: rgb(85, 255, 0);
border: 1px solid gray;
width: 80px;
margin: -56px 0;
border-radius:   23px;
}
</string>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
        <property name="invertedControls">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <layout class="QGridLayout" name="fps_Layout">
        <property name="spacing">
         <number>0</number>
        </property>
        <item row="0" column="1">
         <widget class="QLCDNumber" name="FPS_LCD">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <family>Franklin Gothic Medium</family>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(0, 255, 0, 50);
 border: true;
	border-color: rgb(0, 255, 0);
	color: rgb(0, 255, 0);
    font-family:  &quot;Franklin Gothic Medium&quot;;
    font-size:  40px;</string>
          </property>
          <property name="digitCount">
           <number>3</number>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <widget class="QLCDNumber" name="lcdMsgCounter">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="QLabel" name="label">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Ignored">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(0, 255, 0, 50);
 border: true;
	border-color: rgb(0, 255, 0);
	color: rgb(0, 255, 0);
    font-family:  &quot;Franklin Gothic Medium&quot;;
    font-size:  40px;</string>
          </property>
          <property name="text">
           <string>FPS:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="2">
       <layout class="QGridLayout" name="button_Layout">
        <property name="sizeConstraint">
         <enum>QLayout::SetMaximumSize</enum>
        </property>
        <property name="spacing">
         <number>2</number>
        </property>
        <item row="0" column="1">
         <widget class="myButton" name="buttonLowBeam">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Low Beam</string>
          </property>
         </widget>
        </item>
        <item row="0" column="0">
         <widget class="myButton" name="buttonHighBeam">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>-1</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size:  10px;</string>
          </property>
          <property name="text">
           <string>High Beam</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="myButton" name="buttonReverseLight">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Reverse Light</string>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="myButton" name="buttonTrailerMechanism">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>8</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>Trailer Mechanism </string>
          </property>
         </widget>
        </item>
        <item row="2" column="0" colspan="2">
         <widget class="QLineEdit" name="infoTxt">
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgba(0, 255, 0, 50);
 border: true;
	border-color: rgb(0, 255, 0);
	color: rgb(0, 255, 0);
    font-family:  &quot;Franklin Gothic Medium&quot;;
    font-size:  20px;</string>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
   <zorder>horizontalLayoutWidget</zorder>
   <zorder>tabWidget</zorder>
   <zorder>horizontalLayoutWidget_2</zorder>
   <zorder>checkBoxWheel</zorder>
   <zorder>checkBoxPower</zorder>
   <zorder>reset</zorder>
   <zorder>checkBoxWheel</zorder>
   <zorder>reset</zorder>
   <zorder>horizontalSlider</zorder>
   <zorder>label</zorder>
   <zorder>FPS_LCD</zorder>
   <zorder>lcdMsgCounter</zorder>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>927</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuPo_czenie">
    <property name="title">
     <string>MENU</string>
    </property>
    <widget class="QMenu" name="menuLED">
     <property name="title">
      <string>LOW beam</string>
     </property>
     <addaction name="actionON"/>
     <addaction name="actionOFF"/>
    </widget>
    <widget class="QMenu" name="menuHIGH_beam">
     <property name="title">
      <string>HIGH beam</string>
     </property>
     <addaction name="actionON_2"/>
     <addaction name="actionOFF_2"/>
    </widget>
    <widget class="QMenu" name="menuConfig">
     <property name="title">
      <string>config</string>
     </property>
     <addaction name="actionPORT"/>
     <addaction name="actionADDRESS"/>
     <addaction name="separator"/>
     <addaction name="actionAdd_address"/>
     <addaction name="actionShow_IP_arp"/>
    </widget>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="separator"/>
    <addaction name="menuConfig"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="menuHIGH_beam"/>
    <addaction name="menuLED"/>
    <addaction name="separator"/>
    <addaction name="actionEXIT"/>
    <addaction name="actioncamera"/>
    <addaction name="actionpad"/>
   </widget>
   <addaction name="menuPo_czenie"/>
  </widget>
  <action name="actionConnect">
   <property name="text">
    <string>connect</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>disconnect</string>
   </property>
  </action>
  <action name="actionEXIT">
   <property name="text">
    <string>EXIT</string>
   </property>
  </action>
  <action name="actionON">
   <property name="text">
    <string>ON</string>
   </property>
  </action>
  <action name="actionOFF">
   <property name="text">
    <string>OFF</string>
   </property>
  </action>
  <action name="actionON_2">
   <property name="text">
    <string>ON</string>
   </property>
  </action>
  <action name="actionOFF_2">
   <property name="text">
    <string>OFF</string>
   </property>
  </action>
  <action name="actionPORT">
   <property name="text">
    <string>PORT</string>
   </property>
  </action>
  <action name="actionADDRESS">
   <property name="text">
    <string>ADDRESS</string>
   </property>
  </action>
  <action name="actionAdd_address">
   <property name="text">
    <string>add address</string>
   </property>
  </action>
  <action name="actionShow_IP_arp">
   <property name="text">
    <string>show IP (arp)</string>
   </property>
  </action>
  <action name="actioncamera">
   <property name="text">
    <string>camera</string>
   </property>
  </action>
  <action name="actionpad">
   <property name="text">
    <string>pad</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="1" margin="2"/>
 <customwidgets>
  <customwidget>
   <class>myButton</class>
   <extends>QPushButton</extends>
   <header location="global">mybutton.h</header>
  </customwidget>
  <customwidget>
   <class>myTouchSlider</class>
   <extends>QSlider</extends>
   <header location="global">mytouchslider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="zasoby.qrc"/>
 </resources>
 <connections/>
</ui>
